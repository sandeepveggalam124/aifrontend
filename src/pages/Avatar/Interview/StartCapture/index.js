import React, { useContext } from 'react'
import { Link } from 'react-router-dom'
import { LoginContext } from '../../../../contexts/loginContext'
import { CapitalizeName } from '../../../../utils/capitalizeName'

const StartCapture = () => {
  const {user:name}=useContext(LoginContext)
  const userName=CapitalizeName(name)
  return (
    <>
    <div className='card bg-black'>
        <div className='card-body'>
            <p className='card-text text-center'>
            {userName}, Please be ready to sit in front of your system's camera
            as we'll be capturing pictures of your face from various angles:
             left, right, straight on, top corner, and bottom corner.
            </p>
        </div>
      </div>
      <div>
        <button className='btn btn-light rounded-4 fs-6 fw-bold'><Link to='/interview/captureIcons' className='text-decoration-none text-dark'>Click</Link></button>
      </div>
      </>
  )
}

export default StartCapture