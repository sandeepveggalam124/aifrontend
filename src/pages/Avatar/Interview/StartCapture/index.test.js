import { MemoryRouter as Router } from "react-router-dom";
import {render} from "@testing-library/react";
import StartCapture from "./index";


it('renders correctly',()=>{
    const {container}=render(
        <Router>
            <StartCapture/>
        </Router>
    )
    expect(container).toMatchSnapshot()
})