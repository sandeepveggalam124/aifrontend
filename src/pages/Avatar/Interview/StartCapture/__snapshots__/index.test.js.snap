// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="card bg-black"
  >
    <div
      class="card-body"
    >
      <p
        class="card-text text-center"
      >
        Manjula, Please be ready to sit in front of your system's camera as we'll be capturing pictures of your face from various angles: left, right, straight on, top corner, and bottom corner.
      </p>
    </div>
  </div>
  <div>
    <button
      class="btn btn-light rounded-4 fs-6 fw-bold"
    >
      <a
        class="text-decoration-none text-dark"
        href="/interview/captureIcons"
      >
        Click
      </a>
    </button>
  </div>
</div>
`;
